manifests/prometheus-serviceMonitorKubelet.yaml

```diff
@@ -18,6 +18,19 @@ spec:
     scheme: https
     tlsConfig:
       insecureSkipVerify: true
+ - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    honorLabels: true
+    interval: 30s
+    metricRelabelings:
+    - action: drop
+      regex: container_(network_tcp_usage_total|network_udp_usage_total|tasks_state|cpu_load_average_10s)
```

Comment:
none of these metrics exist on this endpoint so no metric relabeling necessary here